### ISSUE TITLE
The add page button does not have an accessible name

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#1391]
## Bug Report

**Problematic behavior**
The user doesn't know where to find it since it isn't properly labeled.

**Expected behavior/code**
The code in question is this

`<button color="primary" class="sc-34780e96-0 kYJbsY --docs--box-button "><span class="sc-34780e96-0 sc-262648e5-0 iiwUsF foIyfo --docs--icon-bg material-icons-filled">add_box</span></button>`

**Steps to Reproduce**
1. Activate the screen reader NVDA or VoiceOver
2. Tab to the subpage add icon
3. And you will hear "add_box", which has nothing to do with the expected function of the button

**Possible Solution**
Add a label to the button with an attribute `aria-label="Add a subpage"`

**Additional context/Screenshots**
<img width="286" height="71" alt="Image" src="https://github.com/user-attachments/assets/5d6cf314-9fea-424a-b95c-03b6bc1a3783" />
